.ci.yaml

```diff
@@ -2102,6 +2102,17 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: routing_test
 
+  - name: Linux_android spell_check_test
+    bringup: true
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "android", "linux"]
+      task_name: spell_check_test
+    scheduler: luci
```

Comment:
Sorry I should have caught this earlier.
Now everything is with cocoon scheduling, and we should remove the `scheduler: luci`.
I created https://github.com/flutter/flutter/issues/109712 to track the doc update.